examples/echo-server/src/main.rs

```diff
@@ -1,21 +1,17 @@
 use std::process::ExitCode;
-use via::{App, Error, Next, Pipe, Request, Response, Server};
+use via::response::{Finalize, Response};
+use via::{App, Error, Next, Request, Server};
 
 async fn echo(request: Request, _: Next) -> via::Result {
-    request.pipe(Response::build())
+    request.finalize(Response::build())
```

Comment:
Perhaps this could be a default impl of an `into_response()` fn in the `Finalize` trait?

Reply:
I'm going to add this in a separate PR. This makes a ton of sense.